ui/src/api/queries.ts

```diff
@@ -21,7 +21,7 @@ export const validatorQueryOptions = (validatorId: number | string) =>
   queryOptions({
     queryKey: ['validator', String(validatorId)],
     queryFn: () => fetchValidator(validatorId),
-    // staleTime: Infinity,
+    staleTime: 1000 * 60, // every minute
```

Comment:
The default for `staleTime` is `0`, which means it'll refetch in the background whenever:

- New instances of the query mount
- The window is refocused
- The network is reconnected
- The query is optionally configured with a refetch interval

If that leads to too many refetches, `staleTime` is like a time-based throttle. Setting it as low as once-per-minute is effectively not very different from just sticking with the default (leaving it unset).

See: https://tanstack.com/query/latest/docs/framework/react/guides/important-defaults

Reply:
Switching to poll instaead.